forms/user.form.php

```diff
@@ -99,8 +99,15 @@
 		$error['password'] = 'Enter your Password';
 	}
 
+	if (empty($password) || strlen($password) < 8) {
+		throw new InvalidArgumentException("Password cannot be empty and must be at least 8 characters");
```

Comment:
Why use this for error checking?

Reply:
Sorry I don't understand the question...

Reply:
More specifically why use 'throw new InvalidArgumentException' - Just curious how it works.

Reply:
It just gives you more control over how you catch and handle exceptions.